meta-facebook/yv35-rf/src/platform/plat_gpio.c

```diff
@@ -16,59 +16,47 @@ const char *const gpio_name[] = {
 GPIO_CFG plat_gpio_cfg[] = {
 	//  chip,      number,   is_init, is_latch, direction,    status,     property,    int_type,              int_cb
 	/** Group A: 00-07 **/
-	{ CHIP_GPIO, 0, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_LOW, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
-	{ CHIP_GPIO, 1, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_LOW, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
-	{ CHIP_GPIO, 2, ENABLE, DISABLE, GPIO_OUTPUT, GPIO_HIGH, OPEN_DRAIN, GPIO_INT_DISABLE,
-	  NULL },
+	{ CHIP_GPIO, 0, ENABLE, ENABLE, GPIO_OUTPUT, GPIO_LOW, PUSH_PULL, GPIO_INT_DISABLE, NULL },
```

Comment:
cannot be push-pull. It needs to be open-drain output

Reply:
Hi @GoldenBug 
Thank you for your responses.

Considering my testing environment is incomplete (Our RainbowFalls lacks the pm8702). We decided to set the GPIOA0~A2 as the push-pull output type because push-pull is more reliable than open-drain.
In addition, because there are buffers between the AST1030 and the pm8702, we can use the push-pull without considering voltage level differences.

I will test whether setting as the open-drain output type is OK for these three GPIO pins after I get the complete RainbowFalls board. If it is, I will update the GPIO table then.